mdtraj/tests/test_trajectory.py

```diff
@@ -502,8 +502,10 @@ def test():
 
 def test_iterload_skip():
     files = ['frame0.nc', 'frame0.h5', 'frame0.xtc', 'frame0.trr',
-             'frame0.dcd', 'frame0.binpos', 'legacy_msmbuilder_trj0.lh5',
+             'frame0.dcd', 'frame0.binpos',
              'frame0.xyz', 'frame0.lammpstrj']
+    if not (on_win and on_py3):
```

Comment:
i'm not sure you picked up everything from that PR https://github.com/mdtraj/mdtraj/pull/913/files